editor/src/components/shared/scoped-variables.ts

```diff
@@ -131,9 +133,59 @@ function getMatchingElementForVariable(variable: Variable) {
       return jsxElementWithoutUID('span', jsxAttributesFromMap({}), [
         jsExpressionOtherJavaScriptSimple(`JSON.stringify(${variable.name})`, [variable.name]),
       ])
+    case 'array':
+      return jsxFragmentWithoutUID(
+        [jsxTextBlock(`{${variable.name}.map((item) => <span>{JSON.stringify(item)}</span>)}`)],
+        true,
+      )
+    case 'image':
+      return jsxElementWithoutUID(
+        'img',
+        jsxAttributesFromMap({
+          src: jsIdentifierName(variable.name),
+        }),
+        [],
+      )
     default:
       return jsxElementWithoutUID('span', jsxAttributesFromMap({}), [
         jsExpressionOtherJavaScriptSimple(`JSON.stringify(${variable.name})`, [variable.name]),
       ])
   }
 }
+
+function getTypeByValue(value: any): InsertableType {
```

Comment:
This is a minor one, but we try our hardest to avoid `any`, preferring to use the `unknown` type instead, since `any` makes it far to easily punch a hole through the type system without realising that you've just done that